python/lsst/afw/table/idFactory.cc

```diff
@@ -31,23 +30,21 @@ using namespace py::literals;
 namespace lsst {
 namespace afw {
 namespace table {
+namespace {
 
-PYBIND11_PLUGIN(_idFactory) {
-    py::module mod("_idFactory", "Python wrapper for afw _idFactory library");
-
-    py::class_<IdFactory, std::shared_ptr<IdFactory>> clsIdFactory(mod, "IdFactory");
+using PyIdFactory = py::class_<IdFactory, std::shared_ptr<IdFactory>>;
 
-    /* Operators */
-    clsIdFactory.def("__call__", &IdFactory::operator());
+PYBIND11_PLUGIN(idFactory) {
```

Comment:
I didn't think we had a convention, and I'm pretty sure it doesn't matter in terms of code correctness.